src/sentry/api/endpoints/organization_member/details.py

```diff
@@ -54,11 +54,20 @@
 )
 
 
+class TeamRolesSerializer(serializers.Serializer):
+    teamSlug = serializers.CharField(allow_null=True, allow_blank=True)
+    role = serializers.CharField(allow_null=True, allow_blank=True)
+
+
 class OrganizationMemberSerializer(serializers.Serializer):
     reinvite = serializers.BooleanField()
     regenerate = serializers.BooleanField()
-    role = serializers.ChoiceField(choices=roles.get_choices(), required=True)
-    teams = ListField(required=False, allow_null=False)
+    role = serializers.ChoiceField(
+        choices=roles.get_choices(), required=True
+    )  # deprecated, use orgRole
+    orgRole = serializers.ChoiceField(choices=roles.get_choices(), required=True)
+    teams = ListField(required=False, allow_null=False)  # deprecated, use teamRoles
```

Comment:
I take it this endpoint is undocumented? Otherwise we'd want to surface the deprecation in the docs, of course.